### ISSUE TITLE
Form does not contain a submit button

### ISSUE THREAD

**Issue by libremente** [italia/developers.italia.it#704]
# a11y bug
```
Error: This form does not contain a submit button, which creates issues for those who cannot submit the form using the keyboard. Submit buttons are INPUT elements with type attribute "submit" or "image", or BUTTON elements with type "submit" or omitted/invalid.                                                     
   ├── WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2
   ├── #searchModal > div > div > div:nth-child(2) > div > form
   └── <form autocomplete="off" action=""> <div class="form-g...</form>
```

**Comment by 4Rebin** [comment]
I would like to work on this issue.

**Comment by 4Rebin** [comment]
I would like to work on this issue.

**Comment by Trustidiong** [comment]
Hey, I would like to work on this issue. I think it will be fun.

**Comment by mishrajiharsh219** [comment]
Addressing #704
Hey, I would love to work on this, Open-source is awesome.

**Comment by libremente** [comment]
Thanks a lot @4Rebin, waiting forward for your PR!
Thanks also to @mishrajiharsh219, if you want to help us you could review @4Rebin's PR 👍

**Comment by bfabio** [comment]
Hello @4Rebin, are you still working on this?
